### ISSUE TITLE
Allow to counter-sign petetion

### ISSUE THREAD

**Issue by CM63** [fallen/Pytition#78]
Allow some body not to agree the petition and to sign against it, to mean he does not approve it, he want the petition to be rejected.

**Comment by CM63** [comment]
Most of the time, there are much more people who do not approve the petitions than those who approve them. Petition launchers would be surprised about that.

**Comment by fallen** [comment]
This is an interesting idea.
It raises the question of what would mean "signing against" the petition.
I guess it depends on how the petition is written.
If it's a YES/NO question, like "Do you agree on ?" it has the meaning of disagreeing. But it does not give the reason why. You can disagree with someone for lots of reasons. Sometimes you agree on most of the things but just disagree with one point. That would count all disagreement likewise.

Also, if it's not a YES/NO question but more something like "We see this situation, we think this is not normal, we ask <some elected person> to act", what would mean counter-signing? 
* Does it mean the situation is not well described?
* Does it mean the situation is well described but seen as "normal"?
* Does it mean the <elected person> asked to act is not the correct one?
* Does it mean the "plan of action" asked does not seem correct for the person counter-signing?

To sum this up, it raises the question of the interpretation of the "counter signing".

But I reckon that in some cases (where the petition is very close to a poll I would say) this feature would be useful.

I would propose that this be an optional setting of each petition that the petition creator would activate or not.

**Comment by CM63** [comment]
Thank you for your answer. About the fact that this would be optional: obviously, if the launcher of the petition does not want to provide this ability, it would be his own wish. Free to think of it: maybe the launcher fears a failure(1) .

I should say the user would be advised that if he want to counter-sign, it would be a strong choice, he think the petition is just harmful, not only a matter of plan of action etc.

I answer some of your points:
* Does it mean the situation is not well described?
In this case, just don't sign the petition or make your own one.

* Does it mean the situation is well described but seen as "normal"?
In this case, just don't sign the petition.

* Does it mean the  asked to act is not the correct one?
Not so different of the first point

* Does it mean the "plan of action" asked does not seem correct for the person counter-signing?
In this case, he may sign the petition and act in participate in the motion to propose a better plan of action.

I have had the case for my count, when I had at the same time both conditions:
- the petition defend a cause that make no sense for me,
- defending it would be harmful for another one.

Best regards.

(1) Apologies, English is not my native language.
